extensions/authentication-jwt/src/keys.ts

```diff
@@ -34,3 +35,25 @@ export namespace UserServiceBindings {
   export const USER_CREDENTIALS_REPOSITORY =
     'repositories.UserCredentialsRepository';
 }
+export namespace RefreshTokenConstants {
+  export const REFRESH_SECRET_VALUE = 'r3fr35htok3n';
+  export const REFRESH_EXPIRES_IN_VALUE = '216000';
+  export const REFRESH_ISSURE_VALUE = 'loopback4';
+}
+
+export namespace RefreshTokenServiceBindings {
+  export const REFRESH_TOKEN_SERVICE = BindingKey.create<RefreshTokenService>(
+    'services.authentication.jwt.refresh.tokenservice',
+  );
+  export const REFRESH_SECRET = BindingKey.create<string>(
+    'authentication.jwt.refresh.secret',
+  );
+  export const REFRESH_EXPIRES_IN = BindingKey.create<string>(
+    'authentication.jwt.referesh.expires.in.seconds',
+  );
+  export const REFRESH_ISSURE = BindingKey.create<string>(
+    'authentication.jwt.referesh.issure',
+  );
+  export const DATASOURCE_NAME = 'refreshdb';
+  export const REFRESH_REPOSITORY = 'repositories.RefreshTokenRepository';
+}
```

Comment:
More tsdocs are welcome.

Reply:
@raymondfeng I added the docs in the 3rd commit. PTAL